ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -167,6 +179,15 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service
+- as: aws-ipi-efa-pg-f28
```

Comment:
If we keep this job in amd ci, then we need to remove the one in mult-arch upgrade ci.

Reply:
Let's remove the one in multi to keep jobs aligned